students/PeterB/Session 2/Grid.py

```diff
@@ -0,0 +1,32 @@
+plus = '+'
+minus = '- '
+bar = '|'
+space = ' '
+
```

Comment:
I like the idea of naming the characters, so you could change them in one place.

But better tpo use more generic names, like "corner" in stead of "plus".

This way, it would get kind of confusing to do:

plus = "*"

and if you aren't going to change them, why give them a name?